# Author: Adriano dos Santos Fernandes
#Index: src/common/classes/alloc.h
#===================================================================
#RCS file: /cvsroot/firebird/firebird2/src/common/classes/alloc.h,v
#retrieving revision 1.71
diff -u -p -r1.71 alloc.h
--- Firebird-2.5.6.27020-0/src/common/classes/alloc.h.orig	26 Sep 2007 17:48:20 -0000	1.71
+++ Firebird-2.5.6.27020-0/src/common/classes/alloc.h	20 May 2009 02:20:08 -0000
@@ -48,6 +48,7 @@
 #include <stdlib.h> /* XPG: prototypes for malloc/free have to be in
                        stdlib.h (EKU) */
 #endif
+#include <new>
 
 // MSVC does not support exception specification, so it's unknown if that will be correct or not
 // from its POV. For now, use it and disable the C4290 warning.
